services/app-api/utils/reports/index.ts

```diff
@@ -0,0 +1,76 @@
+import { getPossibleFieldsFromFormTemplate } from "../formTemplates/formTemplates";
```

Comment:
nit: we don't use put logic in `index` files anywhere else (I don't think). We use them as folder aggregate files, so that we can more easily import from within the folder (see `app-api/utils/types/index` as an example). Just trying to be consistent